core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/RegionPlanGenerator.scala

```diff
@@ -64,7 +64,7 @@ abstract class RegionPlanGenerator(
   def allocateResource(
       regionDAG: DirectedAcyclicGraph[Region, RegionLink]
   ): Unit = {
-    val resourceAllocator = new DefaultResourceAllocator(physicalPlan, executionClusterInfo)
+    val resourceAllocator = new NaiveResourceAllocator(physicalPlan, executionClusterInfo)
```

Comment:
use a configuration to switch between `DefaultResourceAllocator` and `NaiveResourceAllocator`. We will do more tests in production before we replace it.